tests/functional_test.py

```diff
@@ -86,7 +86,8 @@ def custom_args(request):
 @pytest.mark.run('first')
 def test_package_uninstall():
     """Uninstall tokendito if it is already installed."""
-    proc = run_process([sys.executable, '-m', 'pip', 'uninstall', '-y', 'tokendito'])
+    proc = run_process([sys.executable, '-m', 'pip', 'uninstall', '-q', '-q', '-y', 'tokendito'])
```

Comment:
Shouldnt this be only only 1 -q, i.e. 
-q, --quiet                 Give less output. Option is additive, and can be used up to 3 times (corresponding to WARNING, ERROR, and CRITICAL logging levels).
Wouldnt 2 -q options remove ERROR output?

Reply:
It's deliberate. `-q -q` handles the case where the package to be removed isn't installed, which is possible depending on the test scenario. Other failures are passed back as actual failures.

Reply:
after discussion with Nico this is ok by me as other errors will catch if uninstallation fails.